server/backend/testspg/conftest.py

```diff
@@ -266,3 +264,46 @@ def auth_headers(test_rsa_key):
     token = jwt_utils.create_jwt_token(test_rsa_key)
     headers = jwt_utils.headers(token)
     return headers
+
+
+@pytest.fixture(scope="function")
+def setup_new_user(db_pg_session: Session):
```

Comment:
A convenient fixture to setup a new user in db, copy-paste from 'server/admin_management' test to here at 'server/backend'.